docs/io/index_unofficial.rst

```diff
@@ -0,0 +1,283 @@
+`ctapipe.io.eventfilereader.EventFileReaderFactory`.
+
+The currently existing EventFileReaders,
+`ctapipe.io.eventfilereader.HessioFileReader` and
+`ctapipe.io.unofficial.eventfilereader.TargetioFileReader` can be used as
+templates to create a new EventFileReader for a different data format.
+
+
+Low-Level Reader-Specific Modules
+===========================
+
+A seperate module should be created for the low-level method used for filling
+the event containers for a specific data format. The naming convention for
+this module is "\*io.py", where "\*" is the short-name for the data format.
+For example: `ctapipe.io.hessio` and `ctapipe.io.unofficial.targetio.targetio`.
+
+The low-level module can be a directory of multiple modules, allowing the
+various components involved in the event building (such as containers) to be
+grouped together for a particular data format.
+
+Each data format can be wildly different, therefore these low-level
+modules can appear very different in how they function. This is fine as long
+as these rules are followed:
+
+* The `ctapipe.io.eventfilereader.EventFileReader` calls the low-level methods
+  such that the high-level interface remains the same as other
+  `ctapipe.io.eventfilereader.EventFileReader`.
+* The low-level methods fill the containers with the relevant data, e.g. the
+  waveforms are stored in "data.r0.tel[].adc_samples" (assuming they are
+  waveforms of data level R0, see the next section for an explanation of data
+  levels and see `ctapipe.io.hessio` and
+  `ctapipe.io.unofficial.targetio.targetio` for examples)
+* The unique 'origin' string is added to the meta dictionary of the container
+  so the data source of the events can be obtained by other algorithms, e.g.
+
+  >>> data.meta['origin'] = "targetio"
+
+
+Data Levels (R0/R1/DL0/DL1)
+===========================
+
+It is important to read your events into the correct corresponding event
+container:
+
+* If the file contains raw events, that still require low-level calibration
+  to be useful for offline analysis, then they should be read into the r0
+  container.
+* If the file contains events which have their low-level calibration fully
+  applied to them, then they should be read into the r1 container.
+* If the file contains events which have their low-level calibration fully
+  applied to them AND they are in a proposed format of DL0 (perhaps including
+  a data reduction approach), then they should be stored in the dl0 container.
+* If the file contains events which are already reduced into an extracted
+  charge per pixel, then they should be stored in the dl1 container.
+
+See this link for more details about the `CTA High-Level Data Model
+Definitions SYS-QA/160517
+<https://jama.cta-observatory.org/perspective.req?projectId=6&docId=26528>`_
+(CTA internal).
+
+
+Containers and R1 Calibration
+=============================
+
+This is an optional section. It is included here because the R1 calibration
```

Comment:
According to what I understood about the factory, this is actually optionlly only if one does not want to use the factory, since the first interface, where different file formats *have to agree* is R1. At least this is what I understood. 

So this part, just like the `EventFileReader.is_compatible`method is mandatory in context of the factory. 

Therefore, I would find it good, to discuss both features together.

Reply:
No, it is separate to the `EventFileReaderFactory`, and only needs to be considered by the creator if the data format they are reading will be stored in the `r0` container.